website/docs/test_core_lib.md

```diff
@@ -0,0 +1,187 @@
+---
+id: test_core_lib
+title: Test Core Lib
+sidebar_label: Test Core Lib
+---
+
+Let's understand how `Core-Lib` is initialized and how to integrate it with your new or existing application.
+
+## Main Class
+Here you'll have all the `DataAccess`, `Service`,  `Connection`, `Cache` initialized. Which can be further accessed when we initialize the `Core-Lib`.
+
+`user_core_lib.py`
+```python
+from omegaconf import DictConfig
+
+from core_lib.cache.cache_handler_ram import CacheHandlerRam
+from core_lib.core_lib import CoreLib
+from core_lib.connection.sql_alchemy_connection_registry import SqlAlchemyConnectionRegistry
+
+from user_core_lib.data_layers.data_access.customer_data_access import CustomerDataAccess
+from user_core_lib.data_layers.data_access.user_data_access import UserDataAccess
+from user_core_lib.data_layers.service.customer_service import CustomerService
+from user_core_lib.data_layers.service.user_service import UserService
+
+
+class UserCoreLib(CoreLib):
+    def __init__(self, conf: DictConfig):
+        super().__init__()
+        self.config = conf
```

Comment:
client will be a member of the core-lib
+ 
use the instantiate_config to create it